R/generate_docs.R

```diff
@@ -1,4 +1,4 @@
-spark_location <- "/usr/spark-download/unzipped/spark-3.2.1-bin-hadoop2.7"
+spark_location <- "/usr/spark-download/unzipped/spark-3.3.2-bin-hadoop3"
```

Comment:
@sllynn should this refer to spark.matrix?

Reply:
I can probably put it in an environment variable somewhere 🤔